Framework/PythonInterface/mantid/api/src/FitFunctions/IFunctionAdapter.cpp

```diff
@@ -167,6 +167,7 @@ void IFunctionAdapter::setAttribute(const std::string &attName,
     object value = object(handle<>(getAttributeValue(*this, attr)));
     callMethod<void, std::string, object>(getSelf(), "setAttributeValue",
                                           attName, value);
+    storeAttributeValue(attName, attr);
```

Comment:
Line 172 below `IFunction::setAttribute` is an alias for ``IFunction::storeAttributeValue`, so what is the rationale for the duplicate?